docker/gcp_pudl_etl.sh

```diff
@@ -216,7 +228,6 @@ WRITE_DATAPACKAGE_SUCCESS=0
 CLEAN_UP_OUTPUTS_SUCCESS=0
 DISTRIBUTION_BUCKET_SUCCESS=0
 DEPLOY_EEL_HOLE_SUCCESS=0
-ZENODO_SUCCESS=0
```

Comment:
Note that we will no longer get a Zenodo status directly from the builds. But we will get it from the workflow, and it'll also be sent to the `pudl-deployments` channel.